packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -1,10 +1,11 @@
 %global homedir %{_datadir}/%{name}
 %global confdir config
 
+%{?scl:%global scl_prefix %{scl}-}
```

Comment:
In foreman.spec there's a helpful comment why this is here:
```
# explicitly define, as we build on top of an scl, not inside with scl_package
```

I suspect here there's the same reason

Reply:
Fair -- added